lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -247,6 +248,10 @@ def discover_resources
           @logger.info "  - #{r.id}"
         end
       end
+      if (global = resources.select(&:global?).presence)
+        @logger.info("Detected non-namespaced #{'resouce'.pluralize(global.count)} which will never be pruned:")
```

Comment:
let's print this in yellow, similar to these warnings:

https://github.com/Shopify/kubernetes-deploy/blob/a300fc604ba9c6315179b304287d0272bebb91bd/lib/kubernetes-deploy/deploy_task.rb#L319-L322

Reply:
Typo: `resouce`-> `resource`